Dockerfile

```diff
@@ -15,7 +15,13 @@ RUN mv docker/nginx/default.conf /etc/nginx/conf.d/default.conf
 
 RUN ./manage.py assets build
 
-CMD nginx; gunicorn --timeout 45 --bind unix:/tmp/server.sock wsgi:app --workers 3
+CMD nginx && \
+    env PYTHONPATH=$PYTHONPATH:$PWD gunicorn \
```

Comment:
Oh probably. My shell (fish) actually [doesn't support that syntax](https://fishshell.com/docs/current/faq.html#faq-single-env)